src/components/MakeWidget/MakeWidget.tsx

```diff
@@ -60,15 +69,22 @@ const MakeWidget: FC = () => {
         onChangeTokenClick={() => {}}
         onMaxButtonClick={() => {}}
       />
-      <StyledOrderTypeSelector
-        options={orderTypeSelectOptions}
-        selectedOrderTypeOption={orderScopeTypeOption}
-        onChange={setOrderScopeTypeOption}
-      />
+      <OrderTypeSelectorAndRateFieldWrapper>
+        <StyledOrderTypeSelector
+          options={orderTypeSelectOptions}
+          selectedOrderTypeOption={orderScopeTypeOption}
+          onChange={setOrderScopeTypeOption}
+        />
+        <StyledRateField
+          token1="AST"
+          token2="USDT"
+          rate={new BigNumber("0.01455")}
+        />
+      </OrderTypeSelectorAndRateFieldWrapper>
       {orderType === OrderType.private ? (
         <StyledAddressInput value={address} onChange={setAddress} />
```

Comment:
```suggestion
        <StyledAddressInput value={address} onChange={setAddress} onInfoButtonClick={toggleShowOrderTypeInfo}/>
```

missed this before--the styledIconButton in StyledAddressInput has no onClick.

Reply:
Check!